include/zephyr/rtio/rtio.h

```diff
@@ -1177,6 +1177,8 @@ static inline void rtio_cqe_submit(struct rtio *r, int result, void *userdata, u
 static inline int rtio_sqe_rx_buf(const struct rtio_iodev_sqe *iodev_sqe, uint32_t min_buf_len,
 				  uint32_t max_buf_len, uint8_t **buf, uint32_t *buf_len)
 {
+	ARG_UNUSED(max_buf_len);
+
```

Comment:
This needs to go in the `#else` block since `max_buf_len` _is_ actually used if `CONFIG_RTIO_SYS_MEM_BLOCKS` is enabled.

Reply:
ok I also changed the ARG_UNUSED in `include/zephyr/rtio/rtio.h` then